### PR TITLE
restore setThrottle example from #3742

### PR DESCRIPTION
**Pull request by kevinold**
Restore throttle example missing from merge of #3742

### CODE REVIEW

File: content/api/commands/intercept.md [hunk 1]
```diff
@@ -584,6 +584,34 @@ cy.intercept('/users', (req) => {
...
+You can throttle or delay all incoming responses using a `beforeEach()` in the `cypress/support/index.js` file
+
+```ts
+// Code from Real World App (RWA)
+// cypress/support/index.ts
+import { isMobile } from './utils'
+import './commands'
+// Throttle API responses for mobile testing to simulate real world conditions
+if (isMobile()) {
+  cy.intercept({ url: 'http://localhost:3001', middleware: true }, (req) => {
```
Comment (flotwig) [inline content/api/commands/intercept.md:598]: This won't match, right. shouldn't it be...

```suggestion
  cy.intercept({ url: 'http://localhost:3001/*', middleware: true }, (req) => {
```

File: content/api/commands/intercept.md [hunk 2]
```diff
@@ -584,6 +584,34 @@ cy.intercept('/users', (req) => {
...
+#### Throttle or delay response all incoming responses
+
+You can throttle or delay all incoming responses using a `beforeEach()` in the `cypress/support/index.js` file
+
+```ts
+// Code from Real World App (RWA)
+// cypress/support/index.ts
+import { isMobile } from './utils'
+import './commands'
+// Throttle API responses for mobile testing to simulate real world conditions
```
Comment (flotwig) [inline content/api/commands/intercept.md:596]: it feels confusing to reference these imports from RWA in the example section of these docs, idk, is this necessary/is there a better way to present this?

Comment (kevinold) [reply content/api/commands/intercept.md:596]: Sure, but we have them elsewhere in the guide and want to direct folks from the docs to the RWA for a runnable example.  @amirrustam Might have more thoughts.

**Review by amirrustam — APPROVED** [review]

File: content/api/commands/intercept.md (hunk 2 above)
Comment (flotwig) [reply content/api/commands/intercept.md:596]: Idk why this was marked as resolved and merged with no comment, I think this needs to be revisited still. This is a pretty clunky way to introduce RWA imo.

Comment (amirrustam) [reply content/api/commands/intercept.md:596]: @flotwig I do agree, I have it on the list to discuss and improve in our docs improvement meeting today. I think this was just intermediary add for the use case in time for the Monday release.

Overall, we want some better context for users when we talk about the RWA.

**Pull request merged**